content/doc/book/system-administration/FIPS-140.adoc

```diff
@@ -0,0 +1,54 @@
+---
+layout: section
+---
+ifdef::backend-html5[]
+ifndef::env-github[:imagesdir: ../../resources/managing]
+:notitle:
+:description:
+:author:
+:email: jenkinsci-users@googlegroups.com
+:sectanchors:
+:toc: left
+endif::[]
+
+= FIPS-140
+
+[IMPORTANT]
+====
+The Jenkins community does not actively check Jenkins or Plugins for link:https://csrc.nist.gov/pubs/fips/140-2/upd2/final[FIPS-140] compliance issues.
+====
+
+
+== Jenkins 
+
+It may be possible to run Jenkins in a FIPS-140 compliant manner when the <<managing/system-properties#jenkins-security-FIPS140-COMPLIANCE, compliance flag>> is enabled, and the servlet container, the JVM, and the host OS are all appropriately configured.
```

Comment:
```suggestion
It may be possible to run Jenkins in a FIPS-140 compliant manner when the <<../managing/system-properties#jenkins-security-fips140-compliance, compliance flag>> is enabled, and the servlet container, the JVM, and the host OS are all appropriately configured.
```